sdk/storage/azure-storage-blob-stress/templates/stress-test-job.yaml

```diff
@@ -21,35 +24,32 @@ spec:
           set -exa;
           mkdir -p "$DEBUG_SHARE";
           . /mnt/outputs/.env;
-          export AZURE_HTTP_LOG_DETAIL_LEVEL="HEADERS";
-          export AZURE_LOG_LEVEL="INFO";
-          export APPLICATIONINSIGHTS_INSTRUMENTATION_LOGGING_LEVEL="DEBUG";
-          export APPLICATIONINSIGHTS_ROLE_NAME=storage-{{ .Stress.BaseName }};
           java \
-          -javaagent:BOOT-INF/classes/applicationinsights-agent-3.4.17.jar \
-          -Xms1g \
+          -Dotel.service.name={{ .Release.Name }}-{{ .Stress.BaseName }} \
```

Comment:
Would it be easier to call an external script with these commands? Or does that make it harder to tell what's actually running from the pod if you need to do that?